pyyoutube/client.py

```diff
@@ -144,54 +143,28 @@ def _from_client_secrets_file(self, client_secret_path: str):
         Raises:
             PyYouTubeException: missing required key, client_secret file not in 'web' format.
         """
-        try:
-            secrets_data = None
-
-            with open(client_secret_path, "r") as f:
-                secrets_data = json.load(f)
-
-            # For now only 'web' client_secret files are support,
-            # some 'installed' type files can have missing 'client_secret' key
-            if "web" in secrets_data:
-                secrets_data = secrets_data["web"]
 
-                self.client_id = secrets_data["client_id"]
-                self.client_secret = secrets_data["client_secret"]
+        with open(client_secret_path, "r") as f:
+            secrets_data = json.load(f)
 
-                # Set default redirect to first defined in client_secrets file
-                if (
-                    "redirect_uris" in secrets_data
-                    and len(secrets_data["redirect_uris"]) > 0
-                ):
-                    self.DEFAULT_REDIRECT_URI = secrets_data["redirect_uris"][0]
-
-                return
-
-            else:
-                raise PyYouTubeException(
-                    ErrorMessage(
-                        status_code=ErrorCode.INVALID_PARAMS,
-                        message="Only 'web' client_secret file are supported.",
-                    )
-                )
-
-        except KeyError as ke:
+        # For now only 'web' client_secret files are support,
+        # some 'installed' type files can have missing 'client_secret' key
+        if "web" not in secrets_data:
             raise PyYouTubeException(
                 ErrorMessage(
                     status_code=ErrorCode.INVALID_PARAMS,
-                    message=f"client_secret file is missing required key: '{ke.args[0]}'",
+                    message="Only 'web' type client_secret files are supported.",
                 )
             )
 
-        except Exception:
-            # Not sure if catching any other error and
-            # wrapping it in a PyYouTubeException is a good idea
-            raise PyYouTubeException(
-                ErrorMessage(
-                    status_code=ErrorCode.INVALID_PARAMS,
-                    message="Can not load from client_secret file.",
-                )
-            )
+        secrets_data = secrets_data["web"]
+
+        self.client_id = secrets_data["client_id"]
+        self.client_secret = secrets_data["client_secret"]
```

Comment:
This could raise a `KeyError`, because some 'installed' type secrets that don't have `client_secret` set

Reply:
If not have `client_secret `, the oauth2 flow maybe can not finished. So I think this is necessary parameter.

We can cover this keyError to promot to the user.

Reply:
Should I just let it fail with `KeyError`? 
Or  Raise a  `PyYouTubeException` like this ?
``` Python
if "client_secret" not in secrets_data:
    raise PyYouTubeException(
        ErrorMessage(
            status_code=ErrorCode.MISSING_PARAMS,
            message="file is missing required field 'client_secret'.",
        )
    )

```

Reply:
I think we can catch the `KeyError`,  And maybe the `client_id` also not exists. :)

Reply:
Okey, then I'll quickly make the changes